### ISSUE TITLE
NTR: Skin biopsy: bone formation in dermis and subcutaneous fat (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9324]
New term request Skin biopsy: bone formation in dermis and subcutaneous fat (elthat_130531050806): Heterotopic dermal ossification; osteoma cutis  UMLS: 2674700

**Comment by pnrobinson** [comment]
@MickeySegal
this is almost synonymous with
[Subcutaneous ossification HP:0034282](https://hpo.jax.org/app/browse/term/HP:0034282)
possibly we should add a term Dermal ossification, but the HPO tends to avoid bundled terms such as dermis AND subcutaneous
Let me know please if the current term is sufficient?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK459117/ "ossification that is limited to the dermis and subcutaneous tissues"

Having separate dermal and subcutaneous finding seems like the right approach.

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000071
- label: Heterotopic dermal ossification
- parents: [Abnormal dermis morphology (HP:0034924)](https://hpo.jax.org/app/browse/term/HP:0034924)
- pmid: 
- def: Formation of abnormal, extraskeletal bony tissue in the dermis (i.e., the layer of skin that lies beneath the epidermis and above the subcutaneous layer).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9324
- synonyms:
